### DISCUSSION TITLE
Create a PocketBase Discord Server

### DISCUSSION THREAD

**Discussion by kevmodrome** [pocketbase/pocketbase#548] [Polls]
Discussions is great but realtime can sometimes be a lot more fun! :)

Can help with moderation if you feel like it's not something you want to spend time on. I already do moderation on the Svelte Discord server so I have experience.

**Comment by ganigeorgiev** [comment]
This seems to be a frequent request (https://github.com/pocketbase/pocketbase/issues/185, https://github.com/pocketbase/pocketbase/issues/334, https://github.com/pocketbase/pocketbase/issues/471), but at least for now, I prefer to keep issues and discussions in github because it is public and a lot more convenient for me to moderate and sometimes provide more context by linking to old/similar questions.

Discord is nice but requires users to have registration and from my experience following for some time in the past go-pg - it can get really messy, especially when questions accumulate and there is no one to answer them.

Of course community moderated chatrooms/subreddits/etc. are completely ok with me, but currently I'm barely able to provide support over github/email and adding another support channel would be too much for me to handle.

**Reply by kevmodrome** [reply]
Definitely a fair PoV and I completely understand. We've had a hard time with this on the Svelte Discord. Maybe the community needs to grow a bit bigger before we can introduce realtime.

**Reply by osseonews** [reply]
I might be in the minority, but I find Discord to be terribly confusing when it comes to support. I love it for other uses, but not tech support. The threads are all very disjointed and it's near impossible to find past issues, so I find that everyone just asks similar questions all the time. I think Github discussions are the best solution, so please don't move to discord.

**Reply by kevmodrome** [reply]
You wouldn't necessarily use Discord for support, you could still point users towards Discussions or Issues, but sometimes it's nice to be able to talk about something in real time.

**Reply by brittikbasu** [reply]
I am on the svelte discord but I find GitHub discussions to be better for pocketbase. @ganigeorgiev your answers on here are really helpful in understanding what pocketbase is about and where it’s headed. I think pocketbase Q&A should stay on GitHub. As the community grows, other contributors vetted by you can answer questions here, freeing your time for other things.

**Reply by jm61** [reply]
If not bot on the Svelte Discord, I can't imagine the mess... Today I feel gh discussions pretty rational, Gani replies quickly, it's efficient, clear and right to the point!

**Reply by SirNovi** [reply]
Just to add another perspective: I would love to have a discord. I am quite the programming noob and I would prefer to not use valuable time away from actual product development and instead get answers to simple questions by other users who don't mind to spend a minute or two helping out a newbie. 
Of course the active community needs to be large enough to make it worthwhile.

**Comment by kevmodrome** [comment]
For those of use that like real time, I've created an unofficial pocketbase community discord, you can find it here: https://discord.gg/RwH7GmqMqg

**Comment by silenceofthewans** [comment]
I really really hope that discord doesn't become the primary way for support/questions. It's terrible for finding answers because they don't appear in search engine results and it's one more service to login to. Also github is where the code is and also where the developers already are and gh discussions can be as real time as you want.

**Comment by tigawanna** [comment]
The discussions here are just awesome plus they have SEO and show up when you look up any pocketbase issue
Discord would also be nice for a more peer to peer approach and which maybe the mods could forward any major unanswered queries here and let the low level stuff happen there
Plus , we should all try and make @ganigeorgiev  time easier and not spam him and burn him out so maybe a discord might encourage more people to  offer support to the best of their knowledge

**Comment by Anatoly03** [comment]
I think what could be done is recreating a chat app UI-wise similar to proposed Discord, with the backend made in PocketBase.

**Comment by ralyodio** [comment]
we need a FOSS clone of discord. why is everyone keep moving around to these non-opensource chat platforms? It never makes any sense why we give these companies so much power.
